GDJS/Runtime/InAppTutorialMessage.tsx

```diff
@@ -0,0 +1,186 @@
+          display: 'swap',
+        }
+      )
+        .load()
+        .then((fontFace) => document.fonts.add(fontFace));
+    };
+
+    const getDomElementContainer = (
+      runtimeGame: gdjs.RuntimeGame
+    ): HTMLDivElement | null => {
+      // Find the DOM element container.
+      const domElementContainer = runtimeGame
+        .getRenderer()
+        .getDomElementContainer();
+      if (!domElementContainer) {
+        logger.error('No DOM element container found.');
+        return null;
+      }
+      return domElementContainer;
+    };
+
+    export const displayInAppTutorialMessage = (
+      runtimeGame: gdjs.RuntimeGame,
+      message: string,
+      position: string
+    ) => {
+      const domElementContainer = getDomElementContainer(runtimeGame);
+      if (!domElementContainer) {
+        return;
+      }
+
+      if (_container) return;
+
+      const leftOrRight = position.toLowerCase().includes('right')
+        ? 'right'
+        : 'left';
+      const topOrMiddleOrBottom = position.toLowerCase().includes('top')
+        ? 'top'
+        : position.toLowerCase().includes('middle')
+        ? 'middle'
+        : 'bottom';
+
+      const containerPositionStyle: AbsolutePositionStyle = {};
+      const messageContainerPositionStyle: AbsolutePositionStyle = {};
+
+      if (leftOrRight === 'left') {
+        containerPositionStyle.left = padding;
+        messageContainerPositionStyle.left =
+          topOrMiddleOrBottom === 'top' ? 'calc(100% + 10px)' : '0px';
+      } else {
+        containerPositionStyle.right = padding;
+        messageContainerPositionStyle.right =
+          topOrMiddleOrBottom === 'top' ? 'calc(100% + 10px)' : '0px';
+      }
+      if (topOrMiddleOrBottom === 'top') {
+        containerPositionStyle.top = padding;
+        messageContainerPositionStyle.top = '0px';
+      } else if (topOrMiddleOrBottom === 'middle') {
+        containerPositionStyle.top = '50%';
+        messageContainerPositionStyle.transform =
+          'translateY(calc(-100% - 10px))';
+      } else {
+        containerPositionStyle.bottom = padding;
+        messageContainerPositionStyle.transform =
+          'translateY(calc(-100% - 10px))';
+      }
+
+      _loadFont();
+      _container = (
+        <div
+          id={containerId}
+          style={{ ...styles.container, ...containerPositionStyle }}
+        >
+          <div style={styles.avatarContainer}>
+            <div
+              style={{
+                ...styles.messageContainer,
+                ...messageContainerPositionStyle,
+              }}
+            >
+              {message}
+            </div>
+            <img
+              width={60}
+              height={60}
+              alt="GDevelop mascot red hero"
+              src={redHeroImage}
+            />
+          </div>
+        </div>
+      );
+      // @ts-ignore - TS doesn't acknowledge the assignation above that excludes
+      // the possibility that _container can be null.
```

Comment:
You could try:

const container = (<div ...)
domElementContainer.appendChild(_container = container)

not sure if better, but at least avoids a ts-ignore